externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/session/FlinkSessionImpl.scala

```diff
@@ -64,6 +65,24 @@ class FlinkSessionImpl(
   override def open(): Unit = {
     val executor = fSession.createExecutor(Configuration.fromMap(fSession.getSessionConfig))
 
+    sessionManager.getConf.get(ENGINE_INITIALIZE_SQL).foreach { sql =>
```

Comment:
it's supposed to run only one time after the engine is bootstrapped and before ready to serve connections

Reply:
@pan3793 I just moved the engine-level initial SQL to the engine bootstrap process. The downside is that the engine-level initial SQL would not affect sessions. Is that expected?

Reply:
it's consistent with the current Implementation of the Spark engine.

Additional background: @cfmcgrady proposed a different idea, that `sparkSession.clone` should be called instead of `sparkSession.newSession` so that the newly created session could inherit the changes applied to the bootstrap session. The situation is quite different in Flink since there is no bootstrap session concept, thus I think the current behavior makes sense.